fastlane/Pluginfile

```diff
@@ -7,8 +7,8 @@ group :screenshots, optional: true do
 end
 
 # This comment avoids typing to switch to a development version for testing.
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk'
-# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.1'
+# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.2'
```

Comment:
Note: this change came from https://github.com/wordpress-mobile/WordPress-iOS/pull/18799 which I just approved and merged into the release branch